.github/workflows/ci-android-emulator-tests.yml

```diff
@@ -21,20 +17,13 @@ concurrency:
 
 jobs:
   build-and-run-android-jni-tests:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Undo the changes to this file. It looks like a merge conflict resolved incorrectly.